### PR TITLE
Mochawesome HTML report generator

### PR DESCRIPTION
**Pull request by przemuh**

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -12,9 +12,33 @@
...
+  if (test.state === 'failed') {
+    let item = runnable
+    const nameParts = [runnable.title]
+
+    while (item.parent) {
+      nameParts.unshift(item.parent.title)
+      item = item.parent
+    }
+
+    const fullTestName = nameParts
```
Comment (niemeczek) [inline cypress/support/index.js]: When tests will fail on hook Cypress add such info to the screenshot name, in example "before all hook". In such case it should be also added to the name for example by adding 
`if(runnable.hookName) {
            nameParts.push(``${runnable.hookName} hook``)
        }`

File: cypress/support/index.js [hunk 2]
```diff
@@ -12,9 +12,33 @@
...
+
+  if (test.state === 'failed') {
+    let item = runnable
+    const nameParts = [runnable.title]
+
+    while (item.parent) {
+      nameParts.unshift(item.parent.title)
+      item = item.parent
+    }
+
```
Comment (niemeczek) [inline cypress/support/index.js:33]: When tests will fail on hook Cypress add such info to the screenshot name, in example "before all hook". In such case it should be also added to the name for example by adding:
```suggestion
if(runnable.hookName) {
            nameParts.push(`${runnable.hookName} hook`)
        }
```

Comment (przemuh) [reply cypress/support/index.js:33]: Thanks 👍

**Review by muhilEdvoy — COMMENTED** [review]
its correct

**Pull request closed without merging**